libcloud/storage/drivers/azure_blobs.py

```diff
@@ -214,8 +237,9 @@ def _xml_to_container(self, node):
             'meta_data': {}
         }
 
-        for meta in list(metadata):
-            extra['meta_data'][meta.tag] = meta.text
+        if metadata:
+            for meta in list(metadata):
+                extra['meta_data'][meta.tag] = meta.text
```

Comment:
is meta.tag guaranteed to be not None?

Reply:
Leaving as-is since it's not a required part of this fix.